<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_AllGates</class>
 <widget class="QWidget" name="Widget_AllGates">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>301</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btn_orGate">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>15</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/icon-gate-or.png</normaloff>:/Resources/Button Icons/icon-gate-or.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_recieverGate">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>240</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/icon-gate-input.png</normaloff>:/Resources/Button Icons/icon-gate-input.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_andGate">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>135</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/icon-gate-and.png</normaloff>:/Resources/Button Icons/icon-gate-and.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_notGate">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>15</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/icon-gate-not.png</normaloff>:/Resources/Button Icons/icon-gate-not.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sourceGate">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>335</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/icon-gate-output.png</normaloff>:/Resources/Button Icons/icon-gate-output.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_timerGate">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>370</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/icon-gate-timer.png</normaloff>:/Resources/Button Icons/icon-gate-timer.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_inputOn">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>335</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/icon-gate-input-on.png</normaloff>:/Resources/Button Icons/icon-gate-input-on.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_inputOff">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>370</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/icon-gate-input-off.png</normaloff>:/Resources/Button Icons/icon-gate-input-off.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_GateTriAnd">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>55</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Gates/gate-tri-and.png</normaloff>:/Resources/Gates/gate-tri-and.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_GateTriOr">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>55</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/icon-gate-tri-or.png</normaloff>:/Resources/Button Icons/icon-gate-tri-or.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_numberOutputGate">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>240</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Gates/gate-number-output.png</normaloff>:/Resources/Gates/gate-number-output.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="ln_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>225</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="ln_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>320</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_GateEor">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>95</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Gates/gate-eor.png</normaloff>:/Resources/Gates/gate-eor.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_GateTriEor">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>95</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Gates/gate-tri-eor.png</normaloff>:/Resources/Gates/gate-tri-eor.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="scrollSliderLayout" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>5</y>
     <width>21</width>
     <height>290</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_labelGate">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>280</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string>Label</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="ln_3">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>410</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_nandGate">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>175</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Gates/gate-nand.png</normaloff>:/Resources/Gates/gate-nand.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_norGate">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>135</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Gates/gate-nor.png</normaloff>:/Resources/Gates/gate-nor.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_xorGate">
   <property name="geometry">
    <rect>
     <x>65</x>
     <y>175</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Gates/gate-neor.png</normaloff>:/Resources/Gates/gate-neor.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>scrollSliderLayout</zorder>
  <zorder>btn_orGate</zorder>
  <zorder>btn_recieverGate</zorder>
  <zorder>btn_andGate</zorder>
  <zorder>btn_notGate</zorder>
  <zorder>btn_sourceGate</zorder>
  <zorder>btn_timerGate</zorder>
  <zorder>btn_inputOn</zorder>
  <zorder>btn_inputOff</zorder>
  <zorder>btn_GateTriAnd</zorder>
  <zorder>btn_GateTriOr</zorder>
  <zorder>btn_numberOutputGate</zorder>
  <zorder>ln_1</zorder>
  <zorder>ln_2</zorder>
  <zorder>btn_GateEor</zorder>
  <zorder>btn_GateTriEor</zorder>
  <zorder>btn_labelGate</zorder>
  <zorder>ln_3</zorder>
  <zorder>btn_nandGate</zorder>
  <zorder>btn_norGate</zorder>
  <zorder>btn_xorGate</zorder>
 </widget>
 <resources>
  <include location="../resouces.qrc"/>
 </resources>
 <connections/>
</ui>
